src/p_pspr.c

```diff
@@ -708,7 +708,7 @@ void A_Saw(player_t *player, pspdef_t *psp)
                           linetarget->x, linetarget->y);
 
   if (angle - player->mo->angle > ANG180)
-    if (angle - player->mo->angle < -ANG90/20)
+    if ((signed int) (angle - player->mo->angle) < -ANG90/20)
```

Comment:
This is from Chocolate Doom.

Reply:
This was also my first idea, but since Chocolate Doom didn't trigger this warning, I sneeked how it was solved there.